### PR TITLE
Use redux in react components

### PR DESCRIPTION
**Pull request by coker2019**
- store is added to index.js
- slicer is added to book state
- Button component is added
- event handler is added

no-linters-error
No Linters error on my code editor but linters are not passing here.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <>
+      <header>
+        <NavBar />
+      </header>
+      <main>
+        <Routes>
+          <Route path="/bookSave/" element={<Home />} />
+          <Route path="/bookSave/categories" element={<Categories />} />
+        </Routes>
```
Comment (brytebee) [inline src/App.jsx]: Look at the console of this screen shot below:
image

- The routes are having some conflict/errors, the message means you're asking the router to render a route you haven't defined. Kindly fix it.
- The links only changes the navigation URL but the same page is rendered as the image.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @coker2019 👋,

##

### Highlights💡
- Good project setup.✔️
- Good commit message.✔️ 
- Good PR Summary.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @coker2019 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
image

<h2>You have submitted a project, that fails in linters/ ESLint. Please ensure you have a green tick before requesting a review next time</h2>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by coker2019** [comment]
> ## Invalid Code Review Request ❌
> Hi @coker2019 ,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code. image
> 
> ## You have submitted a project, that fails in linters/ ESLint. Please ensure you have a green tick before requesting a review next time
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Thanks for your response. please I am not seeing any Linters errors from my code editor and its only showing up here, I dunno what to do. I have attached screenshots on my pull request description.
please advise @tugsem

**Comment by coker2019** [comment]
> * store is added to index.js
> * slicer is added to book state
> * Button component is added
> * event handler is added
> 
> no-linters-error No Linters error on my code editor but linters are not passing here.

@tugsem

**Review by divinecharlotte — APPROVED** [review]
# Hello @coker2019      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive pr👍 
- Good implementation of the required changes 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — COMMENTED** [review]
Hi @coker2019  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**